### PR TITLE
Adds challenge field in SubmissionAdmin

### PR DESCRIPTION
**Pull request by ChrsMark**
closes https://github.com/Cloud-CV/EvalAI/issues/1826

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@ChrsMark Can you please paste the screenshot?

**Comment by RishabhJain2018** [comment]
@ChrsMark Can you please resolve the conflicts?

File: apps/jobs/admin.py [hunk 1]
```diff
@@ -7,10 +7,13 @@
```
Comment (RishabhJain2018) [inline apps/jobs/admin.py]: Also, can you please add searching by using `challenge title`.

**Comment by ChrsMark** [comment]
@RishabhJain2018:
- search added
- rebased on top of latest master

**Comment by deshraj** [comment]
Looks like this issue is already fixed and deployed. Hence closing this PR.

**Pull request closed without merging**